webapp/src/features/NurseVisitItem/DischargeNotes.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Row, Col, Card, Accordion } from 'react-bootstrap'
+
+const DischargeNotes = () => {
+	const dischargeData = [
+		{
```

Comment:
Refactor mockData
{
nurseAfterVisitSummary: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
providerAfterVisitSummary:"Lorem ipsum dolor sit amet, consectetur adipiscing elit",
schoolExcuseSelection:"Patient may return to school Today"
}